package.json

```diff
@@ -40,7 +40,8 @@
     "fs-plus": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "standard src spec",
-    "test": "jasmine-focused --captureExceptions spec"
+    "lint": "standard src spec",
```

Comment:
This script isn't going to run in CI, so CI won't run the lint anymore after this PR.

`appveyor.yml` and `.travis-ci.yml` could be updated to run `npm run lint`.

Reply:
Fixed in https://github.com/atom/git-utils/pull/104/commits/4a5e204c5a6992613de06e6a7208ca3a366df77e

Reply:
Thanks.